web/packages/core/src/config.ts

```diff
@@ -58,4 +58,5 @@ export const DEFAULT_CONFIG: Required<Config> = {
     publicPath: null,
     polyfills: true,
     playerVersion: null,
+    preferredRenderer: RenderBackend.WebGpu,
```

Comment:
I think this would better default to `null`, to better indicate that this is subject to change and that the default behavior is to choose the best available renderer.

Reply:
Makes sense! I'll update it accordingly when I get a chance